src/assets/stylesheets/style.css

```diff
@@ -0,0 +1,74 @@
+.balsamiq {
+  font-family: 'Balsamiq Sans', cursive;
+}
+
+h1 {
+  font-size: 4rem;
+  margin-left: 4rem;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 5px;
+}
+
+.scores-section {
+  padding: 0 4em 0 12em;
+}
+
+.input-section {
+  padding: 0 12em 0 4em;
+}
+
+form {
+  display: grid;
+  margin-top: 25px;
+}
+
+.recent {
+  display: flex;
+}
+
+.recent div {
+  flex: 1;
+}
+
+input {
+  height: 40px;
+  margin-bottom: 20px;
+  margin-top: 10px;
+  border: 3px solid black;
+  padding: 0 20px;
+  font-size: 0.98rem;
+}
+
+.btn {
+  width: 6.5rem;
+  height: 2.5rem;
+  font-size: 1.2rem;
+  border: 3px solid black;
+  box-shadow: 5px 5px;
+  border-radius: 7%;
+  cursor: pointer;
+}
+
+.btn-right {
+  float: right;
+}
```

Comment:
- [ ] The submit button should be aligned to the right, together with the form elements as can be seen in the given [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). Kindly ensure to improve upon this aspect.


**Your design:**        |  **The Wireframe design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://user-images.githubusercontent.com/12745474/209071475-f31f7fd6-17c3-4293-8e95-8a50900c4ab5.png"> |  <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/209071597-181f3600-5c6d-493f-a8e7-1aaad3cffc7d.png">

Reply:
Thank you for the correction, Lol didn't notice it, I guess I have a mind for functionality